src/pages/[platform]/build-a-backend/test-notifications/index.mdx

```diff
@@ -29,4 +29,3 @@ import commonRoute from '/src/fragments/lib/push-notifications/common/testing.md
     'react-native': commonRoute
   }}
 />
-mmonRoute }} />
```

Comment:
I'm guessing this might've been just a syntax error from the migration, it looks like the substring from the Fragment component above so I just removed it

Reply:
Nice!